docs/configuration.md

```diff
@@ -10,9 +10,11 @@ Here is an example `.scalafmt.conf`:
 
 ```scala config
 align.preset = more    // For pretty alignment.
-maxColumn = 100 // For my wide 30" display.
+maxColumn = 1234
 ```
 
+> 🚧 Before using specific configuration make sure that your project agrees on the standards. Settings such as `maxColumn` could be a source of issues if different tools such as an IDE uses a different default value.
```

Comment:
I changed the example and made the additional comment about the specific problem users might encounter.